### ISSUE TITLE
Error executing 'ansible-playbook start.yml'

### ISSUE THREAD

**Issue by bqwzhl20** [pingcap/dm#23]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.
2019/01/22 18:57:43.871 main.go:38: [error] parse cmd flags err flag provided but not defined: -log-rotate

2. What did you expect to see?

3. What did you see instead?

4. Versions of the cluster

    - DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):
./dmctl -V
Release Version: v1.0.0-alpha-1-g6b61fa7
Git Commit Hash: 6b61fa79d72d0e388c7e2a8c630a063e47056df0
Git Branch: master
UTC Build Time: 2019-01-21 10:04:51
Go Version: go version go1.11.2 linux/amd64
./dm-master -V
Release Version: v1.0.0-alpha-1-g6b61fa7
Git Commit Hash: 6b61fa79d72d0e388c7e2a8c630a063e47056df0
Git Branch: master
UTC Build Time: 2019-01-21 10:04:57
Go Version: go version go1.11.2 linux/amd64
./dm-worker -V
Release Version: v1.0.0-alpha-1-g6b61fa7
Git Commit Hash: 6b61fa79d72d0e388c7e2a8c630a063e47056df0
Git Branch: master
UTC Build Time: 2019-01-21 10:04:54
Go Version: go version go1.11.2 linux/amd64
        ```
        (paste DM version here, and you must ensure versions of dmctl, DM-worker and DM-master are same)
        ```

    - Upstream MySQL/MariaDB server version:

        ```
        (paste upstream MySQL/MariaDB server version here)
        ```
Server version: 5.6.27-log MySQL Community Server (GPL)

    - Downstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
        (paste TiDB cluster version here)
        ```

    - How did you deploy DM: DM-Ansible or manually?

        ```
        (leave DM-Ansible or manually here)
        ```
       DM-Ansible
    - Other interesting information (system version, hardware config, etc):

        >
        >

5. current status of DM cluster (execute `query-status` in dmctl)

6. Operation logs
   - Please upload `dm-worker.log` for every DM-worker instance if possible
   - Please upload `dm-master.log` if possible
   - Other interesting logs
   - Output of dmctl's commands with problems
   
7. Configuration of the cluster and the task
   - `dm-worker.toml` for every DM-worker instance if possible
   - `dm-master.toml` for DM-master if possible
   - task config, like `task.yaml` if possible
   - `inventory.ini` if deployed by DM-Ansible

8. Screenshot/exported-PDF of Grafana dashboard or metrics' graph in Prometheus for DM if possible

**Comment by csuzhangxc** [comment]
@bqwzhl20 Thanks for your feedback, we have fixed it in #24, and archived a new [dm-v1.0.0-alpha-linux-amd64.tar.gz](https://github.com/pingcap/dm/releases/download/v1.0.0-alpha/dm-v1.0.0-alpha-linux-amd64.tar.gz).

**Comment by bqwzhl20** [comment]
i test it，it's ok
